cg/meta/microsalt/lims.py

```diff
@@ -30,13 +30,23 @@ def get_lims_comment(self, sample_id: str) -> str:
 
         return ""
 
-    def get_organism(self, sample_obj: Sample) -> str:
+    def get_lims_organism(self, sample_id: str) -> str:
```

Comment:
Well you know what I think... Why is this function in two places?

Reply:
are you saying it is also in the lims-api?

Reply:
removed